frontends/precinct-scanner/src/screens/poll_worker_screen.tsx

```diff
@@ -349,6 +385,13 @@ export function PollWorkerScreen({
     }
     setCurrentPollsTransitionTime(timePollsTransitioned);
     updatePollsState(getPollsTransitionDestinationState(pollsTransition));
+    await logger.log(
```

Comment:
I realize I'm being a little silly here but can we add a message in the polls_opened case that states that there are zero ballots scanner, or add as a seperate param the ballotsScanned count. Just so we make sure we are extra explicit in the success case that we have checked this.

Reply:
I don't think it would hurt to include the ballotsScanned count on all poll transition logs so that might be easiest.

Reply:
Done